### PR TITLE
Add tests for some plugins

### PR DESCRIPTION
**Pull request by GaetanLepage**

### CODE REVIEW

File: plugins/utils/emmet.nix
Comment (traxys) [inline plugins/utils/emmet.nix:1]: Unrelated, please do a standalone PR

**Review by traxys — CHANGES_REQUESTED** [review]

File: plugins/utils/emmet.nix
Comment (GaetanLepage) [reply plugins/utils/emmet.nix:1]: Done !

**Review by traxys — APPROVED** [review]
